scene/gui/popup_menu.cpp

```diff
@@ -454,10 +454,10 @@ void PopupMenu::_input_from_window_internal(const Ref<InputEvent> &p_event) {
 		Input *input = Input::get_singleton();
 		Ref<InputEventJoypadMotion> joypadmotion_event = p_event;
 		Ref<InputEventJoypadButton> joypadbutton_event = p_event;
-		bool is_joypad_event = (joypadmotion_event.is_valid() || joypadbutton_event.is_valid());
+		is_gamepad_event = (joypadmotion_event.is_valid() || joypadbutton_event.is_valid());
```

Comment:
"joypad" is how it's named in the API, I don't think this local variable rename is useful.

Reply:
Nevermind, I see other code was already using "gamepad_event" here and you made this a member, so I agree with that name for local consistency.